docs/user-guide.md

```diff
@@ -498,6 +539,46 @@ You can export and import prompts.
 
 ![](./img/import-export-prompt.png)
 
+### Publish
+
+You can publish prompts to make them available for the target audience. You can also unpublish them to withdraw from the public use.
+
+**Note**: to publish a shared prompt, [duplicate](#duplicate-1) it and then publish. 
+
+**To publish a prompt**: 
+
+1. Click **Publish** in the prompt's menu. 
+
+![](./img/publish-prompt.png)
+
+2. In the pop-up window:
+   * Enter **publication request name** in the upper area.
+   * In **Publish to**, select the organization or a folder to publish your prompt into.
```

Comment:
'Publish to' indicates only where we put the publication, i.e. the destination folder. We define the target audience in 'allow access...'